lib/utils.js

```diff
@@ -266,11 +272,14 @@ module.exports.getModuleChunk = getModuleChunk;
  * @param {Object} options
  * @param {string} options.chunkName
  * @param {string} options.context
+ * @param {string} options.fullHash
  * @param {string} options.content
  * @return {string}
  */
 function interpolateSpriteFilename(filename, options) {
-  return filename.replace(/\[chunkname\]/g, options.chunkName);
+  return filename
+    .replace(/\[hash(:\d)?]/g, getSizedMatch.bind(this, options.fullHash))
```

Comment:
yes, but only for [hash] pattern, all filename related replacements has no sense